static/app/utils/theme/theme.tsx

```diff
@@ -161,12 +161,13 @@ const generateThemeUtils = (
   colors: ReturnType<typeof deprecatedColorMappings>,
   aliases: Aliases
 ) => ({
-  tooltipUnderline: (underlineColor: ColorOrAlias = 'gray300') => ({
+  tooltipUnderline: (underlineColor: ColorOrAlias | string = 'gray300') => ({
```

Comment:
@JonasBa there’s only a handful of usages that use `tooltipUnderline` with a specific color, and it’s only:

- `linkUnderline`
- `gray200`
- `background`
- `success`
- `successText`, `warningText`, `errorText`
- `green300`, `red300`, `gray300`

I think we should be able to find token replacements for those and just use our color tokens here?